q2_types/sample_data/_format.py

```diff
@@ -7,19 +7,50 @@
 # ----------------------------------------------------------------------------
 
 import qiime2.plugin.model as model
+from qiime2.plugin import ValidationError
 
 from ..plugin_setup import plugin
 
 
 class AlphaDiversityFormat(model.TextFileFormat):
-    def sniff(self):
+    def _validate_(self, level):
         with self.open() as fh:
-            i = 0
-            for line, i in zip(fh, range(10)):
-                cells = line.split('\t')
-                if len(cells) < 2:
-                    return False
-            return i > 1
+            header = None
+            records_seen = 0
+            file_ = enumerate(fh) if level == 'min' else zip(range(10), fh)
```

Comment:
```suggestion
            file_ = enumerate(fh, 1) if level == 'min' else zip(range(1, 10), fh)
```

Reply:
This line becomes 81 cols long (which was why I didn't do this in the first place). Options: promote from a ternary to a real if/else block; unindent the whole block by removing the filehandle context manager and instead manually closing. Any other options come to mind?